src/main/java/com/ss/mqtt/broker/handler/packet/in/SubscribeInPacketHandler.java

```diff
@@ -15,5 +19,14 @@
     protected void handleImpl(@NotNull UnsafeMqttClient client, @NotNull SubscribeInPacket packet) {
         var ackReasonCodes = subscriptionService.subscribe(client, packet.getTopicFilters());
         client.send(client.getPacketOutFactory().newSubscribeAck(packet.getPacketId(), ackReasonCodes));
+        ackReasonCodes.forEach(client, SubscribeInPacketHandler::checkException);
```

Comment:
if you have for example two SHARED_SUBSCRIPTIONS_NOT_SUPPORTED reason codes, you will send a disconnect packet twice